presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -520,6 +474,22 @@ public Object validateClusteringKey(Object value)
         }
     }
 
+    public static boolean containsUnsupportedType(DataType dataType)
```

Comment:
make it package private
```suggestion
   static boolean containsUnsupportedType(DataType dataType)
```